### PR TITLE
Updated for affected since

### PR DESCRIPTION
**Pull request by qjhart**

### CODE REVIEW

File: harvest/experts-client/bin/experts-cdl.js [hunk 1]
```diff
@@ -168,10 +194,15 @@ program.name('cdl-profile')
   .description('Import CDL Researcher Profiles and Works')
   .option('--source <source...>', 'Specify linked data source. Used instead of --fuseki')
   .option('--output <output>', 'output directory')
+  .option('--username <username>', 'Specify CDL username')
```
Comment (qjhart) [inline harvest/experts-client/bin/experts-cdl.js:197]: @rakunkel-ucd Are these two lines used?

File: harvest/experts-client/lib/query/work/bind.rq [hunk 2]
```diff
@@ -28,9 +28,10 @@ WHERE {
       ("software" false ucdlib:Work "software" "")
       ("thesis-dissertation" false ucdlib:Work "thesis" "dissertation")
     }
+  # typo fixed on 34
```
Comment (qjhart) [inline harvest/experts-client/lib/query/work/bind.rq:31]: rm comment

File: harvest/experts-client/package.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "@comunica/query-sparql": "^2.7.1",
     "@rdfjs/fetch-lite": "3.2.1",
     "@ucd-lib/experts-api": "^1.1.0",
+    "chalk": "^5.3.0",
```
Comment (qjhart) [inline harvest/experts-client/package.json:30]: Required?

File: harvest/experts-client/bin/experts-cdl.js (hunk 1 above)
Comment (rakunkel-ucd) [reply harvest/experts-client/bin/experts-cdl.js:197]: No

File: harvest/experts-client/lib/query/work/bind.rq (hunk 2 above)
Comment (rakunkel-ucd) [reply harvest/experts-client/lib/query/work/bind.rq:31]: sure

File: harvest/experts-client/package.json (hunk 3 above)
Comment (rakunkel-ucd) [reply harvest/experts-client/package.json:30]: no

**Pull request merged**